src/bbs2gh/Program.cs

```diff
@@ -47,7 +47,7 @@ public static async Task Main(string[] args)
             }
             catch (Exception ex)
             {
-                Logger.LogWarning("Could not retrieve latest gei CLI version from github.com, please ensure you are using the latest version by running: gh extension upgrade ado2gh");
+                Logger.LogWarning("Could not retrieve latest gei CLI version from github.com, please ensure you are using the latest version by running: gh extension upgrade bbs2gh");
```

Comment:
```suggestion
                Logger.LogWarning("Could not retrieve latest bbs2gh extension version from github.com, please ensure you are using the latest version by running: gh extension upgrade bbs2gh");
```

Reply:
Fixed in a2c57ef014d977a45328d459db90aabee09bf801!